susemanager/susemanager.spec

```diff
@@ -161,6 +162,13 @@ BuildRequires:  docbook-utils
 %description tools
 This package contains SUSE Manager tools
 
+%package bash-completion
+Summary:        Bash completion for SUSE Manager CLI tools
```

Comment:
Are we going to call it in this way also on Uyuni?

Reply:
Good point. Do we have a trick to set the product name dynamically in the description/summary? Or do we need to differentiate it between branches?

Reply:
Looks like we use "Spacewalk" invariably in most spec files. But the other packages in `susemanager.spec` uses "SUSE Manager" for Uyuni as well.

Reply:
We do not have different package names for SUMA and Uyuni. Only for the GPG keys we have it different. But there only because we trust different keys.
And as this is a sub-package, we should not make it more complicated. Just use susemanager-bash-completion is totally fine for me